test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpRuntimeTests.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Web;
+using System.Web.Caching;
+using System.Web.Hosting;
+using AutoFixture;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests;
+
+public class HttpRuntimeTests
+{
+    private readonly Fixture _fixture;
+
+    public HttpRuntimeTests()
+    {
+        _fixture = new Fixture();
+
+        // Note: Uncommenting the following lines fixes the issue
+        //
+        //var servicesCollection = new ServiceCollection();
+        //servicesCollection.AddTransient<Cache>();
+
+        //HostingEnvironmentAccessor.Current = new HostingEnvironmentAccessor(
+        //    new DefaultServiceProviderFactory().CreateServiceProvider(servicesCollection),
+        //    Options.Create(new SystemWebAdaptersOptions())
+        //);
```

Comment:
@twsouthwick This fixes the runtime exception but is not a great solution. I'm happy to extend the `HostingEnvironmentAccessor` and/or add a factory to set the `HostingEnvironmentAccessor.Current` but unsure what's the best approach.